src/vivarium_census_prl_synth_pop/components/synthetic_pii.py

```diff
@@ -140,9 +140,7 @@ def name(self):
 
     def setup(self, builder: Builder):
         super().setup(builder)
-        self.clock = (
-            builder.time.clock()
-        )  # todo: change to start time since yob only goes to 2020 for data
+        self.clock = builder.time.clock()
```

Comment:
Removing unnecessary comment and formatting.

Reply:
NameGenerator class will be deleted from the synthetic pii module following post-processing of last names.